dev-tools/audius-compose

```diff
@@ -310,6 +320,12 @@ def up(
 
     AAO_DIR = pathlib.Path(os.getenv("AAO_DIR", protocol_dir / "../anti-abuse-oracle"))
 
+    localhosts = "127.0.0.1       ingress audius-protocol-storage-1 audius-protocol-storage-2 audius-protocol-storage-3 audius-protocol-storage-4 audius-protocol-comms-discovery-1 audius-protocol-comms-discovery-2 audius-protocol-comms-discovery-3 audius-protocol-discovery-provider-1 audius-protocol-creator-node-1 audius-protocol-creator-node-2 audius-protocol-creator-node-3 audius-protocol-identity-service-1 audius-protocol-solana-test-validator-1 audius-protocol-eth-ganache-1 audius-protocol-poa-ganache-1"
+    hosts_file = pathlib.Path("/etc/hosts")
+    if localhosts not in hosts_file.read_text():
+        print("Adding development hosts to /etc/hosts")
+        hosts_file.write_text(hosts_file.read_text() + "\n" + localhosts + "\n")
+
```

Comment:
Can you format this better?

Reply:
Also, have backup for more than 3 nodes

Reply:
can it stay as a big wall of text? I think it's easier to remove and mange in the future if it's just one line instead of like 20 lines for all the services. I added the other nodes for backup now tho

Reply:
Multiline string can be processed later, if anything, keep it is a file outside of the python code, that is probably best practice and easiest to see

Reply:
moved to formatted constant ✅